test/net/sourceforge/kolmafia/request/coinmaster/SkeletonOfCrimboPastRequestTest.java

```diff
@@ -54,4 +65,39 @@ void exitsTheShopChoice() {
       assertPostRequest(requests.getFirst(), "/choice.php", "whichchoice=1567&option=5");
     }
   }
+
+  @Test
+  public void canRegisterDailySpecial() {
+    // Daily Special: <item description link> (10 knucklebones)
+    String responseText =
+        "Daily Special: <a class=\"nounder\" href=\"descitem(352309634)\"><b>rethinking cnady</b></a> (10 knucklebones)";
+
+    // Visit the skeleton with the response text
+    SkeletonOfCrimboPastRequest.visit(responseText);
+
+    // Verify preferences are set correctly
+    assertThat(
+        Preferences.getInteger("_crimboPastDailySpecialItem"), is(ItemPool.RETHINKING_CANDY_BOOK));
+    assertThat(Preferences.getInteger("_crimboPastDailySpecialPrice"), is(10));
+    assertThat(
+        Preferences.getBoolean("_crimboPastDailySpecial"),
+        is(false)); // false means "not yet bought"
+  }
+
+  @Test
+  public void dailySpecialPopulatesInConcoctionPool() {
```

Comment:
Perhaps it might be better to test this more end-to-end. It simply being in the concoction pool doesn't verify that "create" works properly